### DISCUSSION TITLE
Object detection for only one camera and detection is not correct

### DISCUSSION THREAD

**Discussion by luxapana** [blakeblackshear/frigate#4434] [Ask A Question]
Hi 
I've configured frigate with three IP cameras. Object detection for persons is on and only using CPU for the moment. I can see CPU usage at 200% for the detector.
The issue is out of three cameras, only one is triggering detections. Even in this case, it detects my dog as a person with 70% score. 

Please see attached the config and a thumbnail of a detection.
What could be causing these issues?

Thanks
[config.txt](https://github.com/blakeblackshear/frigate/files/10043646/config.txt)
![Screenshot 2022-11-18 231757](https://user-images.githubusercontent.com/45680862/202769314-ef32c856-8f03-4029-9999-898a467a7926.png)

**Comment by NickM-27** [comment]
By default frigate only tracks person. You need to configure Frigate to track other objects. 

As far as why only one is detecting, that may be related if it doesn't see a person.

**Reply by luxapana** [reply]
It is already configured to detect Persons's not dogs. I've stimulated other two cameras with persons infront, but they are not detected.

**Reply by NickM-27** [reply]
I see, you'll want to read https://docs.frigate.video/guides/false_positives for reducing false positives.

You can watch the debug view in frigate webUI to see what the camera seems. Enable motion and bounding boxes to see what is detected. 

What device are you running on? Seems it may be overwhelmed.

**Reply by luxapana** [reply]
Thanks, will check that out.
I can see now that one of the other cameras detected a person accurately. However, it definitely misses many. Perhaps my old i5 laptop is overloaded and misses events? Please see below for the debug view.

![image](https://user-images.githubusercontent.com/45680862/202775946-20882ccc-7ef2-46cd-ab3d-8699355aed75.png)

**Reply by NickM-27** [reply]
You'll see skipped fps (only during the time an object is visible) which indicates it isn't processing everything. 

Other than that, watching the camera debug view is helpful with bounding boxes enabled (cameras -> open camera -> choose debug)
